lib/frrevent.h

```diff
@@ -223,7 +223,15 @@ static inline unsigned long timeval_elapsed(struct timeval a, struct timeval b)
 	}) /* end */
 
 /* Prototypes. */
-extern struct event_loop *event_master_create(const char *name);
+
+/*
+ * event_master_create
+ * inputs:
+ * name - The name of the event system that is being created
+ * limit_fds - Ignore the systems built in limit and use 1000 or
+ *             the actual value which ever is lesser
+ */
+extern struct event_loop *event_master_create(const char *name, bool limit_fds);
```

Comment:
gosh, I'd rather we not change this one api like this

Reply:
fixed